tracer/test/snapshots/Security.AspNetCore2.__test=discovery.scans_enableSecurity=True_expectedStatusCode=404_url=_Health_login.php.verified.txt

```diff
@@ -22,7 +22,7 @@
       network.client.ip: 127.0.0.1,
       runtime-id: Guid_1,
       span.kind: server,
-      _dd.appsec.json: {"triggers":[{"rule":{"id":"nfd-000-001","name":"Detect common directory discovery scans","tags":{"category":"attack_attempt","type":"security_scanner"}},"rule_matches":[{"operator":"match_regex","operator_value":"^404$","parameters":[{"address":"server.response.status","highlight":["404"],"key_path":[],"value":"404"}]},{"operator":"phrase_match","operator_value":"","parameters":[{"address":"server.request.uri.raw","highlight":["/login.php"],"key_path":[],"value":"http://localhost:00000/Health/login.php"}]}]}]},
```

Comment:
This looks like it's just an ordering change in the keys? Can we add a scrubber to verify to order these consistently so there's no changes, and no future changes?

Reply:
Thanks, good catch. We were actually sorting the keys before, but a regression in scrubbers broke that. I fixed and am able to remove loads of files from this PR.